common/app/common/commercial/CommercialProperties.scala

```diff
@@ -9,7 +9,8 @@ import play.api.libs.json.Json
 
 case class CommercialProperties(
   editionBrandings: Set[EditionBranding],
-  editionAdTargetings: Set[EditionAdTargeting]
+  editionAdTargetings: Set[EditionAdTargeting],
+  prebidIndexSites: Option[Set[PrebidIndexSite]]
```

Comment:
Is there a reason that this is an `Option[Set[...]]` rather than just being a `Set` and being default `Set.empty` (like `editionalAdTargeting` seems to be)?

Reply:
It's to avoid problems when fronts get pressed.  If it isn't optional, the front will throw a 500 until it's re-pressed.  For the same reason, it's easier to change it or remove it as well.
I can't remember why this isn't a problem with `editionAdTargetings`.

Reply:
Ah okay - if it is given a default of `Set.empty` would that work? (asking mainly out of curiosity).

Reply:
Only if you modified the json encoding so that an empty set doesn't get included in the pressed json at all.
The problem is when it comes to read the json something like this will be expected for an empty set:
`prebidIndexSites: []`
but there will actually be nothing there at all.